### ISSUE TITLE
How to download the data directly from the BRENDA database?

### ISSUE THREAD

**Issue by diegovalenzuelaiturra** [Gangl2016/Optimal-growth-temperture-predictor#1]
Incredible work!

It would be really helpful if you could share the scripts to make the queries and download the data from BRENDA since its API seems to be broken and I still could not figure out how to solve it.

Thanks!

**Comment by mengqvist** [comment]
Yes, their API is a bit tricky. It can be made to work with their SOAP client. What part of the data is it that you're looking to get?
